message.module

```diff
@@ -306,7 +307,6 @@ function template_preprocess_message(&$variables) {
   $variables['teaser'] = $variables['view_mode'] == 'teaser';
   $variables['message'] = $variables['elements']['#message'];
 
-  /** @var \Drupal\message\MessageInterface $message */
```

Comment:
I think it's pretty clear that this should be the `Message` object in there. There is no need in this comment.